### ISSUE TITLE
New Commands for Support Channels

### ISSUE THREAD

**Issue by datphotogurl** [OperationCode/operationcode-pybot#325]
### Create New Command Rules for Support Channels
There are two private support channels: #operation-pride and #Blacks-In-Tech on OC's Slack. A suggested command is needed so new and existing members can safely request to join these channels.

1. Create a `/pride` command that allows the member to click "Do you want to join the #operation-pride channel? Click the `Request` button to request to join or `Cancel`" so an automatic request notifies the channel #operation-pride that a new member wants to join.
1a. Create a notification once the member clicks the `Request` button that their request has been sent.

2. Create a `/blacks-in-tech` command that allows the member to click "Do you want to join the #Blacks-In-Tech channel? Click the `Request` button to request to join or `Cancel`" so an automatic request notifies the channel #Blacks-In-Tech that a new member wants to join.
2a. Create a notification once the member clicks the `Request` button that their request has been sent.

**Comment by JudsonStevens** [comment]
This functionality will be very similar to the functionality of the greeter response or the mentor request response.

**Comment by aaron-suarez** [comment]
@JohnathanBaldera I'm going to be working on this on Friday, Dec 17th. If you'd like to join, let me know via slack. If you're not in our slack workspace, sign up at https://operationcode.org/join and you'll receive an invitation. I'm `@raz0r` on there

**Comment by JudsonStevens** [comment]
Closing this as it's been handled in the rewrite - we added in a more general functionality that can apply to any channel join request. For these specifically we included two new slash commands - `/join-pride` and `/join-blacks-in-tech` that send requests to the channel with an interactive button for someone to hit when/if they decide to invite the requestor. This also includes action tracking through threading. The below screenshot shows both pieces of functionality. We also send the user a notification that their request has been received.
<img width="494" alt="Screen Shot 2022-01-01 at 8 49 17 AM" src="https://user-images.githubusercontent.com/35241250/147853222-5f7e8edf-dd6b-4158-b403-92988af5ac0a.png">
<img width="707" alt="Screen Shot 2022-01-01 at 8 50 00 AM" src="https://user-images.githubusercontent.com/35241250/147853246-4a51f5da-496e-4947-9cf9-0afdc2455a98.png">
